sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -187,6 +187,15 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val VERIFY_STATS_FROM_FILESYSTEM_WHEN_BROADCASTJOIN =
```

Comment:
This config name implies it only does verification when broadcast join. However, seems that it verifies the statistics no matter doing broadcast join or not.